downstream/assemblies/platform/assembly-multi-machine-install.adoc

```diff
@@ -9,13 +9,6 @@ ifdef::context[:parent-context: {context}]
 [role="_abstract"]
 You can use these instructions to install {PlatformName} as multiple {ControllerName} nodes and {HubName} with an external managed database.
```

Comment:
Not something you edited, but if possible, could we erase the self reference and go to the main verb:

"You can install {PlatformName} as multiple {ControllerName} nodes and {HubName} with an external managed database."